### ISSUE TITLE
Question: website admins in the warehouse

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#892]
When I send out notifications that the warehouse is going to be offline, I email this to all website admins, based on the admins listed in the users_websites table.

I've just realised that for about a third of the c. 150 websites listed, there is no admin role specified, which means that these websites will not receive any notifications of planned downtime.

Do we have any other sources of contact information for these websites?

**Comment by johnvanbreda** [comment]
@BirenRathod don't you have an email contact list for this purpose? If we can cross reference this with the list of website admins and find any gaps then it's quite likely I'll have contact details for some of them.

**Comment by BirenRathod** [comment]
@kitenetter has already got that list.

**Comment by kitenetter** [comment]
I did have a list from @BirenRathod quite a few years ago, but as far as I recall the only updates that I've made to our contacts list since then are based on the admins listed in the users_websites tables. Some of the websites that don't have an admin listed have been added to the warehouse quite recently, so won't be on the original list from Biren. (Some of the older websites may well be redundant now, or were only for testing/demo purposes, and can be ignored.)

@BirenRathod I will email you a list of the websites that are missing admins to see if any contact names can be added.

@johnvanbreda if there are still any gaps after that I may need to come back to you for some of the recent additions.
